### PR TITLE
set up program for reading in NC ipums

### PR DESCRIPTION
**Pull request by maggiesu0725**
Hi Leah,

I modified the programs for reading in ACS 1 year data for 2017 plus the vacancy file. Are there any other steps I'm missing for reading in the data? vacancy file is still downloading so the file name haven't been updated yet. Is there a reason we were reading in the vacancy data separately?

@lhendey

### CODE REVIEW

**Comment by maggiesu0725** [comment]
Will add all other years once 2017 is finalized. @lhendey

**Comment by lhendey** [comment]
@maggiesu0725 - the original files call the hud_inc_all macro which has values for the DC region and shouldn't be applied to North Carolina. I would comment that out and add a statement that the limits aren't calculated. As part of the other programs in the NCHsg folder - i would dowdload the excel files for the income limits (see for example https://www.huduser.gov/portal/datasets/il.html#2017_data) - merge on the limits to the appropriate county and calculate the categories. make sense?

**Comment by maggiesu0725** [comment]
@lhendey I think there's only a 30% AMI one that's in excel. Do we want to calculate the other thresholds based on this number? (e.g. 120% AMI would be 4 times 30%AMI, not accurate because 120% is from 50% but 50% and 30% are not perfectly proportional )? Or somehow scrape everything on the website?

**Comment by lhendey** [comment]
@maggiesu0725 - no they are all there. look for  "Data for Section 8 Income Limits in MS EXCEL" in the first section.

**Comment by rpitingolo** [comment]
@maggiesu0725 I re-downloaded the raw data. Please try using usa_00119.dat for the 2017 program. Thanks!

**Comment by maggiesu0725** [comment]
Could you help redownload for year 2012 as well? usa_00118.dat in raw folder is for 2010. @rpitingolo  Thanks!

**Comment by rpitingolo** [comment]
@maggiesu0725 usa_00120.dat is 2012 data.

**Comment by maggiesu0725** [comment]
@lhendey @rpitingolo program for reading in NC ACS 1 year data 2012-2017 are ready for review. Then I can proceed on to the tabulations.

File: Prog/ACS_2012_NC.sas [hunk 1]
```diff
@@ -0,0 +1,750 @@
...
+    otherwise do;
+      %err_put( msg="Invalid state code: " _n_= serial= statefip= pernum= )
+    end;
+  end;
+  
+  drop i;
+
+run;
+
+proc contents data= ACS_2017_NC;
```
Comment (lhendey) [inline Prog/ACS_2012_NC.sas]: remove this proc contents (for2017 - causing error and one runs with the finalize macro). - i think this error is on the other programs, please correct.

**Review by lhendey — COMMENTED** [review]
@maggiesu0725 - can you move the Hud_inc programs to ipums/prog/temp on your d drive? that way  we aren't adding them to the master before they are ready.  

there is one rogue proc contents on some of the programs that's throwing an error. please remove before finalizing.

**Comment by maggiesu0725** [comment]
@lhendey please review.

**Review by lhendey — APPROVED** [review]

**Pull request merged**